snapshotter/service_integ_test.go

```diff
@@ -54,7 +54,7 @@ func TestLaunchContainerWithRemoteSnapshotter_Isolated(t *testing.T) {
 
 	vmID := 0
 
-	testTimeout := 300 * time.Second
+	testTimeout := 1 * time.Minute
```

Comment:
go test has its own timeout. Do we really need the timeouts in the tests?

Reply:
Guess not. The default is fine for both this test and the `MultipleContainers` test on local runs. So it's fine to just remove them.